src/main/java/de/mediathekview/mserver/crawler/orf/tasks/OrfHelper.java

```diff
@@ -0,0 +1,55 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.crawler.orf.OrfConstants;
+import java.util.ArrayList;
+import java.util.List;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+/**
+ * Helper methods for ORF tasks
+ */
+public class OrfHelper {
+  
+  private static final String LETTER_URL_SELECTOR = "div.mod_name_list > ul.js_extra_content > li:not(.inactive) > a.base_list_item_inner";
+  
+  public static String parseTheme(final Element aItem) {
+    String theme = aItem.attr(Consts.ATTRIBUTE_TITLE);
+    
+    // Thema steht vor Doppelpunkt
+    // Ausnahmen 
+    // - ZIB-Sendungen mit Uhrzeit
+    // - DokEins-Sendungen
+    // - Ungarisches Magazin
+    int index = theme.indexOf(":");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around ':' to use the faster "indexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)